neuro_data/pipeline_management/NeuroDataPipelineManagement.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import datajoint as dj
+
+
+from neuro_data.static_images.data_schemas import StaticScanCandidate, StaticScan, ConditionTier, Frame, InputResponse, Eye, Treadmill, StaticMultiDataset, StaticMultiDatasetGroupAssignment, ExcludedTrial
```

Comment:
this one is optional but we usually import the module (`from neuro_data.static_images import data_schemas`) and then use the fully quialifiied name `data_schemas.StaticScanCandidate`, that allows us to know right away what classes are defined where

Reply:
Normally I design my stuff where there is 1 class per module, not a bunch of them in one. Hence I tend to follow the format of C++ design